sympy/geometry/plane.py

```diff
@@ -777,21 +787,106 @@ def projection(self, pt):
             return rv
         return self.intersection(Line3D(rv, rv + Point3D(self.normal_vector)))[0]
 
-
     def random_point(self, seed=None):
-        """ Returns a random point on the Plane.
+        """ Returns a random point on the Plane somewhere within
```

Comment:
I think it'd be more natural to have a random point that could in principle be anywhere on the plane, not just in a part of it. The normal distribution, `random.gauss`, could be used for the parameters u, v here: then the result is a bivariate Gaussian distribution covering the entire plane.

Reply:
Currently, the line entity takes another approach, uniformly distributing the random points over an arbitrarily chosen large intervals (something with `2**32`). I think using Gaussian in both cases would be better; it's by far the best known probability distribution with unbounded support.

Reply:
That makes sense. I'll give it a try for both Line and Plane.